recommendation/microsoft.owin.json

```diff
@@ -160,6 +160,36 @@
                 "Contains": "public",
                 "NotContains": "override"
               }
+            },
+            {
+              "Name": "AddExpression",
+              "Type": "Class",
+              "Value": "RequestDelegate _next = null; //CTA Change",
+              "Description": "Add a new global variable to replace the OwinMiddleware global abstract variable of Next.",
+              "ActionValidation": {
+                "Contains": "RequestDelegate_next=null;//CTAChange",
```

Comment:
replace `// CTA Change` with `// PortingAssistant change`

Reply:
Sorry you already mentioned these. Those fixes are already included in non-merged rules CR which would have overwritten these older changes in the merge. I went ahead and updated them anyway however. Thanks.

Reply:
Ah, ok. Wasn't aware of that!